portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -123,13 +123,66 @@ definition {
 	}
 
 	@description = "Verify that a Help Text can be added"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132199 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+            fieldPositionNumber = "0",
+            fieldType = "Single Selection",
+            formPageNumber = "0");
+
+		var fieldSingleSelectionField = FormFields.getFieldName();
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
```

Comment:
Hi @leilanyaragao ,
I'd like to ask you to use `FormFields.editOptionsRow` to edit a Single Selection option.

In this scenario, you would use something like this to edit the first option:

```
FormFields.editOptionsRow(			
			optionName = "Option1",			
			optionValue = "Option 1");
```

You can find an example [here](https://github.com/liferay/liferay-portal/blob/cc5ccd2a3e82780cc87de95c93973ee585d632ab/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L1821-L1828).